### ISSUE TITLE
Strange korean translation on NTP customization widget

### ISSUE THREAD

**Issue by simonhong** [brave/brave-browser#12898]
I think `Brave Together` would be better than `Brave 함께`. I think only `Together` is partly tranlated.
For Top Sites, `Top` is translated to `최고의`. However, its meaning is much close to highest, greatest.
In our case, its meaning should be close to`most visited` instead.
and `자주 방문한 사이트`(which means `Most visited sites`) is better for for `Top Sites` translation. Or just using `탑 사이트` (pronounced Top site in korean).

<img width="741" alt="스크린샷 2020-11-26 오전 9 48 57" src="https://user-images.githubusercontent.com/6786187/100295543-32ae0a80-2fcd-11eb-947f-043bd3b7f855.png">

**Comment by simonhong** [comment]
@mkarolin Let's fix this also together :)

**Comment by simonhong** [comment]
Changed `Top Sites` translations from transifex.

**Comment by simonhong** [comment]
`Brave 함께` is changed to `Brave Together` from transifex.

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.19.87 Chromium: 88.0.4324.96 (공식 빌드) (x86_64)
-- | --
개정 | 68dba2d8a0b149a1d3afac56fa74648032bcf46b-refs/branch-heads/4324@{#1784}
OS | macOS 버전 10.15.7(빌드 19H15)
```

Confirmed updated translation for `Top Sites`

`1.19.86` | `1.19.87`
----- | -----
<img width="746" alt="1 19 86 Korean 2" src="https://user-images.githubusercontent.com/28145373/106007869-bf844b80-6084-11eb-8ccc-9b8bae50a58c.png"> | <img width="749" alt="1 19 87 Korean 2" src="https://user-images.githubusercontent.com/28145373/106007874-c0b57880-6084-11eb-8712-3671de541694.png">

Note - Brave Together card is not available on 1.19.x:
`1.19.86` | `1.19.87`
----- | -----
<img width="761" alt="1 19 86 Korean" src="https://user-images.githubusercontent.com/28145373/106007973-dd51b080-6084-11eb-8c2e-44aaf6b9080b.png"> | <img width="715" alt="1 19 87 Korean" src="https://user-images.githubusercontent.com/28145373/106007978-ddea4700-6084-11eb-9242-91023ac82ae4.png">
